pkg/template/api/v1/types.go

```diff
@@ -14,15 +14,23 @@ type Template struct {
 	// Standard object's metadata.
 	kapi.ObjectMeta `json:"metadata,omitempty"`
 
-	// Objects is an array of objects to include in this template. Required.
+	// message is an optional instructional message that will
+	// be displayed when this template is instantiated.
```

Comment:
> our default client is new-app and it's not going to ignore it.

As opposed the `oc process` command we built specifically to instantiate templates?